.rat-excludes

```diff
@@ -63,14 +63,15 @@ logs
 .*dependency-reduced-pom.xml
 known_translations
 json_expectation
-local-1422981759269/*
-local-1422981780767/*
-local-1425081759269/*
-local-1426533911241/*
-local-1426633911242/*
-local-1430917381534/*
+local-1422981759269
+local-1422981780767
+local-1425081759269
+local-1426533911241
+local-1426633911242
+local-1430917381534
 local-1430917381535_1
 local-1430917381535_2
+local-1425081759268/*
```

Comment:
what does this mean? Shouldn't we just remove this line and delete the old logs that we're not even using anymore?